app/scripts/directives/blocks.js

```diff
@@ -251,6 +251,14 @@ angular
               return campusNames.data;
             });
         };
+        if ($scope.answer.value) {
+          $scope.searchingCampuses = true;
+          $scope.searchCampuses($scope.answer.value).then((data) => {
+            if (data.length === 0) {
```

Comment:
In JS, `0` is `false`, so if the length is 0 run line 258, if 1 or above skip it.
```suggestion
            if (!data.length) {
```